prometheus-exporters/install-exporter-amd64.sh

```diff
@@ -220,7 +220,7 @@ Description=Prometheus Nginx Exporter
 
 [Service]
 User=root
-ExecStart=/usr/local/bin/nginx-prometheus-exporter -nginx.scrape-uri=http://localhost/basic_status
+ExecStart=/usr/local/bin/nginx-prometheus-exporter -nginx.scrape-uri=http://localhost:8080/basic_status
```

Comment:
I think we should change `basic_status` to `stub_status`. I'm trying to align with the default uri mentioned in the exporter's documentation:

```
  -nginx.scrape-uri string
        A URI or unix domain socket path for scraping NGINX or NGINX Plus metrics.
        For NGINX, the stub_status page must be available through the URI. For NGINX Plus -- the API. The default value can be overwritten by SCRAPE_URI environment variable. (default "http://127.0.0.1:8080/stub_status")
```